### DISCUSSION TITLE
UI Project Is Moving

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#81] [General]
UI Project is moving to a new repository

The current UI has gotten to large to maintain in this repository

I am not a UI developer and unfortunately the UI is slowing down development work of the project, a small backend change is now usually accompanied by hours of updates to the UI

I have decided to revert the UI back to a pure example only application, this will speed up development of new core features.

The current feature rich UI will be moved to a new repo and will still be worked on, but will be based on the nuget releases of OnnxStack, so will be a week or 2 behind feature wise and may not have a weekly release schedule like OnnxStack

I will Link to the new UI repo once its ready :)

**Comment by TheyCallMeHex** [comment]
For those who are specifically interested in the OnnxStack Windows UI App, we want to clarify its purpose. Originally conceived as an sample project like the Console app, the UI app gradually evolved with each release into a more comprehensive product. However, it was never intended to take precedence over the core focus of the OnnxStack project.

In order to realign our development goals, we have streamlined the UI project within OnnxStack, returning it to its foundational elements. Simultaneously, we have initiated a distinct and independent project called Amuse. Unlike the UI app, Amuse is designed to be a fully-featured product, operating separately from OnnxStack. By establishing Amuse as its own standalone project, powered by OnnxStack, we aim to provide a platform for the development of unique features that may not necessarily align with the core objectives of the OnnxStack project.

We appreciate your understanding and support as we make these adjustments to enhance the overall development experience.

A new discord for the Amuse project is available here: https://discord.gg/zXE9DWBCCt

The new github for the Amuse project is available here: https://github.com/Stackyard-AI/Amuse

The V1.0.0 release of Amuse is available here: https://github.com/Stackyard-AI/Amuse/releases/tag/v1.0.0
